pkg/helm/chartproxy/testdata/RepoIndexPreRelease.yaml

```diff
@@ -0,0 +1,78 @@
+      keywords:
+        - amd64
+        - deploy
+        - Catalog
+        - Commercial
+        - Limited
+        - Other
+        - RHOCP
+      kubeVersion: '>=1.20.0-beta.2'
```

Comment:
Usually a helm chart that support pre-release versions specifies kubeVersion range with a `-0` at the end. Below is a helm chart metadata from our catalog that support pre-release version - 

```yaml
- apiVersion: v2
    appVersion: 10.0.0
    created: "2020-11-03T14:58:13.7979+05:30"
    dependencies:
    - alias: sch
      name: ibm-sch
      repository: '@sch'
      version: 1.2.15
    description: |-
      IBM Order management Software Enterprise Edition v10 provides cross-channel order orchestration capabilities to enable intelligent brokering of orders across many disparate systems. It also provides a global view of inventory across the supply chain and enables business users to make changes to order process.
      Documentation
      For additional details regarding install parameters check http://ibm.biz/oms-helm-readme.
      License
      By installing this product you accept the license terms http://ibm.biz/oms-license and http://ibm.biz/oms-apps-license.
    digest: 47f7bae2c23ceec955c497ddd3f1d2adeea08238363a849c3ffc0ebefb9125ff
    home: http://ibm.biz/oms-home
    icon: https://raw.githubusercontent.com/IBM/charts/master/logo/ibm-oms-logo.png
    keywords:
    - oms
    - sterling
    - yantra
    - order
    - fulfillment
    - om
    - amd64
    - ppc64le
    - framework
    - Commercial
    - RHOCP
    - Other
    kubeVersion: '>=1.16.0-0'
    maintainers:
    - name: IBM
    name: ibm-oms-ent-prod
    type: application
    urls:
    - https://redhat-developer.github.com/redhat-helm-charts/charts/ibm-oms-ent-prod-5.1.0.tgz
    version: 5.1.0
```

Reply:
Added https://github.com/openshift/console/pull/7012/commits/81e25d75997565ad4514d6dc224def24290044d9
cc: @rohitkrai03